test/v1/models/audit/CreateBroughtForwardLossAuditDetailSpec.scala

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.audit
+
+import play.api.libs.json.Json
+import support.UnitSpec
+import v1.models.errors.{RuleInvalidLossAmount, RuleTypeOfClaimInvalid}
+
+class CreateBroughtForwardLossAuditDetailSpec extends UnitSpec {
+
+  val nino    = "ZG903729C"
+  val lossId = "lossId"
+
+  "writes" must {
+    "work" when {
+      "success response" in {
+        val json = Json.parse(s"""{
+            |    "userType": "Agent",
+            |    "agentReferenceNumber":"012345678",
+            |    "nino": "$nino",
+            |    "request": {
+            |      "typeOfLoss":"self-employment",
+            |      "selfEmploymentId":"X2IS12356589871",
+            |      "typeOfClaim":"carry-forward",
+            |      "taxYear": "2019-20",
+            |      "lossAmount": 100
+            |    },
```

Comment:
It obviously doesn't matter functionally but, so as not to confuse should use remove the typeOfClaim field (and below in the other test).